src/components/Legend/Legend.tsx

```diff
@@ -16,9 +16,13 @@ export const Legend = ({ context, position = "top", alignment = "left" }: Props)
 	return (
 		<Element
 			className={cx(
+				"absolute",
 				"flex",
 				"gap-2",
-				(position === "left" || position === "right") && "flex-col",
+				position === "right" && "flex-col top-0 bottom-0 right-0",
+				position === "left" && "flex-col top-0 bottom-0 left-0",
+				position === "bottom" && "bottom-0 left-0 right-0",
+				position === "top" && "top-0 left-0 right-0",
```

Comment:
Sorry this has to be non-absolutely positioned.
The reason for this is later we will have more components above/below/left/right of the graph and we want these components to respect the layout of others in that area.
I.E "Above legend"

I can see why you've done it though so iv'e left a comment on the line that would need to change in order for this not to be absolute

Reply:
Okey,
I remove that style and now is alright.
I pushed change.

Something was not right for me that's way I go with absolute styles...
CSS(scss, tailwind) is not my strong skill but I think I will learnt bit doing this project.
From where is coming variable graph is mysterious form me in SCSS rule: grid-area: graph;